task/htmlTask.js

```diff
@@ -4,8 +4,8 @@ const DONE_TASK_BUTTON_TEXT = '\u{02713}';
 const DELETE_TASK_BUTTON_TEXT = "X";
 
 export class htmlTask extends Task {
-    constructor(text = "", done = false, onUpdate, onDelete, onDone, onUnDone) {
-        super(text, done, onUpdate, onDelete, onDone, onUnDone);
+    constructor(text = "", done = false, eventHandlers) {
+        super(text, done, eventHandlers);
```

Comment:
Extract the specific events you expect to receive from `eventHandlers`, and inject the relevant events to `super`

Reply:
done